doc/sources/guide/packaging-osx.rst

```diff
@@ -103,76 +98,62 @@ Complete guide
     $ brew reinstall --build-bottle sdl2 sdl2_image sdl2_ttf sdl2_mixer
 
    .. note::
-       If your project depends on GStreamer or additional libraries (re)install
-       them with ``--build-bottle`` as described below.
-
-#. Install additional libraries::
-
-    $ brew reinstall --build-bottle gstreamer gst-plugins-{base,good,bad,ugly}
-
-   .. note::
-       If your Project needs Ogg Vorbis support be sure to add the
-       ``--with-libvorbis`` option to the command above.
+       If your project depends on GStreamer or other additional libraries (re)install
+       them with ``--build-bottle`` as described `below <additional libraries_>`_.
 
-    Python from Homebrew currently also needs the following patch for GStreamer::
+#. Install Cython and Kivy::
 
-     $ brew reinstall --build-bottle https://github.com/cbenhagen/homebrew/raw/patch-3/Library/Formula/gst-python.rb
+    $ pip install -I Cython==0.21.2
+    $ USE_OSX_FRAMEWORKS=0 pip install git+https://github.com/kivy/kivy.git@1.9.0
 
 #. Install the development version of PyInstaller which includes fixes for the
    GStreamer hooks::
 
     $ pip install git+https://github.com/pyinstaller/pyinstaller.git@develop
 
+#. Export the ``HOOKSPATH`` environment variable::
```

Comment:
Is the env var also used by pyinstaller, or just by this example? In the latter case, I would find it better to avoid this indirection and explain that the hook already comes with pyinstaller, but optionally, a custom one can be used, like the one shipped with kivy.